apps/.storybook/head.html

```diff
@@ -2,4 +2,5 @@
 <link rel="stylesheet" media="all" href="css/common.css">
 <link rel="stylesheet" media="all" href="css/applab.css">
 <link rel="stylesheet" media="all" href="css/droplet/droplet.min.css">
+<script src="https://use.fontawesome.com/8c9a5ecdc4.js"></script>
```

Comment:
I looked for one and amazingly didn't find one. Not sure how it's getting loaded in the first place. Somewhere deep inside some ruby code I imagine.